frontend/tests/unit/Feed.spec.js

```diff
@@ -1,13 +1,51 @@
 import { expect } from 'chai'
-import { shallowMount } from '@vue/test-utils'
-import Feed from '@/components/Feed'
-
-describe('Feed.vue', () => {
-  it('renders props.title when passed', () => {
-    const title = 'new message'
-    const wrapper = shallowMount(Feed, {
-      propsData: { title }
+import { mount, RouterLinkStub } from '@vue/test-utils'
+import Feed from '@/components/course/Feed'
+import ReviewCard from '@/components/reviews-replies/ReviewCard'
+import QuestionCard from '@/components/questions-answers/QuestionCard'
+
+describe('Feed.vue Reviews', () => {
+  before(function() {
+    this.items = [
+      { id: 1, title: 'hey', body: 'wow', likes: 2, published: new Date() },
```

Comment:
Technically these props are wrong now for date. Doesn't seem to break things tho.

Reply:
i've fixed all the tests now